spec/models/user_spec.rb

```diff
@@ -6,9 +6,44 @@
   before :each do
     @user = create :user, :admin
   end
+
   describe 'dispatcher?' do
     it 'gets the not of admin' do
       expect(@user.dispatcher?).to eql false
     end
   end
+
+  describe 'can_delete?' do 
+    context 'when admin' do 
+      it 'returns true' do 
+        item = create :log_entry
+        expect(@user.can_delete?(item)).to eql true
```

Comment:
For `true` and `false` (and a few other things) you can write `to be` rather than `to eql`. Source: https://github.com/umts/custom-cops/blob/master/lib/umts-custom-cops/be_matcher_for_non_duplicable_types.rb